### ISSUE TITLE
ClearAll() not working for FormPickerMultiCheckBoxElement

### ISSUE THREAD

**Issue by dhaksddj** [TheJuki/KFormMaster#30]
## Description

when I call ClearAll() and then select the element, the dialog is already preset with the previous values.
### Expected behavior

no presets set unless value was set in definition of element.

### Actual behavior

Previously user-chosen values are set in the dialog

### KFormMaster version

* Version: 4.0.0

**Comment by TheJuki** [comment]
```
formBuilder.clearAll()
        val multiCheckBox = formBuilder.getFormElement<FormPickerMultiCheckBoxElement<List<ListItem>>>(MultiItems.ordinal)
        multiCheckBox.reInitDialog(this, formBuilder)
```

I will try saving the AlertDialog.Builder to the model so that calling reInitDialog is not necessary. For now, you will need to call reInitDialog(this, formBuilder) after clearing the FormPickerMultiCheckBoxElement.

**Comment by TheJuki** [comment]
I just released [4.0.1](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if clearAll() works for you. You should not need to call reInitDialog() afterward anymore.

**Comment by TheJuki** [comment]
I am closing this issue. If you have any issues feel free to reopen.
